TrailBlaze/Objects/Interception.m

```diff
@@ -0,0 +1,54 @@
+//
+//  Interception.m
+//  TrailBlaze
+//
+//  Created by Adam Issah on 7/26/22.
+//
+
+#import "Interception.h"
+#import "MapKit/Mapkit.h"
+@import Parse;
+
+
+@implementation Interception
+@dynamic rendezvous;
+@dynamic polylineCoords;
+@dynamic startTime;
+@dynamic receiver;
+
++ (nonnull NSString *)parseClassName {
+    return @"Interception";
+}
+
++ (void) uploadRequest: (PFGeoPoint *) rendezvous polyline: (MKPolyline *)polyline receiver: (PFUser *) receiver withCompletion: (PFBooleanResultBlock _Nullable)completion {
```

Comment:
looks like the passed `completion` block is never called?

Reply:
I believe for the parse completion block it automatically sets the succeeded boolean value depending on whether the server handshake was successful of not so succeed still holds a result even if we don't set one with the completion(item, item) function.